extensions/notebook/package.json

```diff
@@ -477,6 +486,10 @@
           "command": "books.sqlserver2019",
           "when": "view == providedBooksView"
         },
+        {
+          "command": "books.hybridjupyterbook",
+          "when": "view == providedBooksView"
```

Comment:
This should be able to be added to the package.json of your extension - adding it here is going to be a problem since this isn't checking whether your extension is installed so will error out if it isn't.